src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -1428,7 +1430,10 @@ private void determineIfCapitalCanBeHeld(final List<ProTerritory> prioritizedTer
     final Territory myCapital = ProData.myCapital;
 
     // Determine max number of defenders I can purchase
```

Comment:
pls update this comment to explain what is happening, that we are placing all units available for purchase in the capitol. The reason that the infra factory has to be non-mobile does not make sense to me. Perhaps some maps will have mobile factories that could boost production in the capitol if they are within range (and even more so if they are in the same tile already as the capitol).

this method otherwise is very hard to follow since it using in/out parameters. Returning a boolean value would make the logic much easier to deduce.